modules/oe_webtools_laco_service/src/LacoServiceHeaders.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_webtools_laco_service;
+
+/**
+ * A simple class that contains the headers used by the Laco service.
+ */
+class LacoServiceHeaders {
```

Comment:
This is a good use case to declare the class as `final`. It's a common thing when classes only contain constants.